frontend/packages/dev-console/src/components/MetricsPage.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { Helmet } from 'react-helmet';
+import { match as RMatch } from 'react-router';
+import { TechPreviewBadge } from '@console/shared';
+import { QueryBrowserPage } from '@console/internal/components/monitoring/metrics';
+import { NamespaceBar } from '@console/internal/components/namespace';
+import { withStartGuide } from '@console/internal/components/start-guide';
+import DefaultPage from './DefaultPage';
+
+export interface MetricsPageProps {
+  match: RMatch<{
+    ns?: string;
+  }>;
+}
+
+const MetricsPage: React.FC<MetricsPageProps> = ({ match }) => {
+  const namespace = match.params.ns;
+  return (
+    <React.Fragment>
+      <NamespaceBar />
+      <Helmet>
+        <title>Metrics</title>
+      </Helmet>
+      {namespace ? (
+        <QueryBrowserPage namespace={namespace} />
+      ) : (
+        <DefaultPage badge={<TechPreviewBadge />} title="Metrics">
+          Select a project to view metrics
+        </DefaultPage>
+      )}
+    </React.Fragment>
+  );
+};
+
+export default withStartGuide(MetricsPage);
```

Comment:
@kyoto I just noticed that you need to wrap the inner component with `withStartGuide` otherwise the namespace bar will be _below_ the start guide header.